framework/examples/myproject/smoke_chip_ingress_test.go

```diff
@@ -48,13 +47,29 @@ func TestChipIngressSmoke(t *testing.T) {
 
 		err := chipingressset.DefaultRegisterAndFetchProtos(ctx, client, []chipingressset.ProtoSchemaSet{
 			{
-				Owner:         "smartcontractkit",
-				Repository:    "chainlink-protos",
+				URI:           "https://github.com/smartcontractkit/chainlink-protos",
 				Ref:           "95decc005a91a1fd2621af9d9f00cb36d8061067",
 				Folders:       []string{"workflows"},
 				SubjectPrefix: "cre-",
 			},
 		}, out.RedPanda.SchemaRegistryExternalURL)
 		require.NoError(t, err, "failed to register protos")
 	})
+
+	t.Run("local protos can be registered", func(t *testing.T) {
+		ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
+		defer cancel()
+
+		createTopicsErr := chipingressset.CreateTopics(ctx, out.RedPanda.KafkaExternalURL, []string{"cre"})
+		require.NoError(t, createTopicsErr, "failed to create topics")
+
+		err := chipingressset.DefaultRegisterAndFetchProtos(ctx, nil, []chipingressset.ProtoSchemaSet{
+			{
+				URI:           "file://../../../../chainlink-protos", // works also with absolute path
```

Comment:
Do we have to mention this in our docs? https://smartcontractkit.github.io/chainlink-testing-framework/framework/components/chipingresset/chip_ingress.html?highlight=chip#chip-ingress-set
Not sure if it's obvious that people can use `file://`

Reply:
added to docs, thanks for catching that!